VerifyReplication/index.js

```diff
@@ -109,6 +110,10 @@ const destinationRequestWorkers = (DST_MD_REQUEST_WORKERS
     && Number.parseInt(DST_MD_REQUEST_WORKERS, 10)) || defaults.DESTINATION_MD_REQUEST_WORKERS;
 const compareObjectSize = COMPARE_OBJECT_SIZE === '1';
 const compareObjectAllVersions = COMPARE_OBJECT_ALL_VERSIONS === '1';
+const delimiter = SRC_DELIMITER || defaults.DELIMITER;
+const prefixes = SRC_BUCKET_PREFIXES ? SRC_BUCKET_PREFIXES.split(',')
+    .filter(x => x)
+    .map(x => (x.endsWith(delimiter) ? x : `${x}${delimiter}`)) : [];
```

Comment:
I wonder if we want this `map` or if it would be better to stick with AWS semantics and not append a delimiter if there is none, trusting the original bucket prefix to be what the user wants?